diff --git a/helm/whereabouts/templates/_helpers.tpl b/helm/whereabouts/templates/_helpers.tpl
index 6ce2d579..4d01118f 100644
--- a/helm/whereabouts/templates/_helpers.tpl
+++ b/helm/whereabouts/templates/_helpers.tpl
@@ -51,6 +51,7 @@ helm.sh/chart: {{ include "whereabouts.chart" . }}
 app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
 {{- end }}
 app.kubernetes.io/managed-by: {{ .Release.Service }}
+io.giantswarm.application.team: {{ index .Chart.Annotations "io.giantswarm.application.team" | quote }}
 {{- end }}

 {{/*
